Assignment-2/Ques3.js

```diff
@@ -0,0 +1,14 @@
+
+// Write a function expression that takes in another function as an argument
+
+
+function getName(){
+    return "Somil" ;
+}
+
+function greetPerson(name){
+
+    return("Welcome "+ name);
+}
+
```

Comment:
Code is not indented. Dont call function from console.. Remove console log. Check code